src/fabric/test/fabric2_doc_crud_tests.erl

```diff
@@ -449,12 +449,19 @@ conflict_on_update_with_no_rev({Db, _}) ->
     ?assertThrow(conflict, fabric2_db:update_doc(Db, Doc2)).
 
 
-conflict_on_create_as_deleted({Db, _}) ->
+allow_create_new_as_deleted({Db, _}) ->
     Doc = #doc{
         id = fabric2_util:uuid(),
         deleted = true,
         body = {[{<<"foo">>, <<"bar">>}]}
     },
+    {ok, {1, Rev}} = fabric2_db:update_doc(Db, Doc),
+    ?assertEqual({not_found, deleted}, fabric2_db:open_doc(Db, Doc#doc.id)),
+    Doc1 = Doc#doc{
+        revs = {1, [Rev]}
+    },
+    ?assertEqual({ok, Doc1}, fabric2_db:open_doc(Db, Doc#doc.id, [deleted])),
+    % Only works for new deleted documents
```

Comment:
Minor nit, but how about "Only works when the document has never existed to match CouchDB 3.x behavior"

Reply:
That sounds better